.github/workflows/linux.yml

```diff
@@ -113,6 +113,10 @@ jobs:
           gem install --version 4.0.7 public_suffix
           gem install --version 2.7.6 dotenv
           gem install --no-document fpm
+      - name: Install linter
+        run: go install github.com/mgechev/revive@v1.3.7
```

Comment:
This might be an opportunity to add a `internal/tools/tools.go` package with `go generate` lines, so the specific version(s) of Go tools can be kept in go.mod and updated with normal dependency updates.

Reply:
Also: maybe add `staticcheck`?